pkg/mount/raw_mount.go

```diff
@@ -74,7 +75,7 @@ func (rm *rawMounter) Load(rawVolumeDevicesPaths []*regexp.Regexp) error {
 	}
 
 	// try to find all bind mounts of raw volumes
-	if len(rawVolumeDevicesPaths) == 0 {
+	if len(rawVolumeDevicesPaths) == 0 || rawVolumeDevicesPaths[0].String() == "" {
```

Comment:
have we seen cases where the path is an empty string? 
Also, do we control want's appended into the array?

Reply:
No, I'm just reverting this line to its original behavior:
https://github.com/libopenstorage/openstorage/commit/697a083511fa246b55de0a6b27825546afd2ef6a#diff-10e7242386af4f87436db51f5a2021fd060dfb7dcb730698f54e9d3b82c191b6L76

Yes, only some pre-defined strings are passed in here from PX